Storage/Storage/Tools/StorageType+Extensions.swift

```diff
@@ -444,6 +444,20 @@ public extension StorageType {
         return allObjects(ofType: TaxClass.self, matching: predicate, sortedBy: nil)
     }
 
+    /// Returns a single TaxRate given a `siteID` and `id`
+    ///
+    func loadTaxRate(siteID: Int64, id: Int64) -> TaxRate? {
+        let predicate = \TaxRate.siteID == siteID && \TaxRate.id == id
```

Comment:
Should we update the parameter to be more descriptive?
```suggestion
    func loadTaxRate(siteID: Int64, taxRateID: Int64) -> TaxRate? {
        let predicate = \TaxRate.siteID == siteID && \TaxRate.id == taxRateID
```

Reply:
Sure, done in fa2180b32003bc05e885d2298f5e9d524ece021a